ui/components/app/beta-header/index.js

```diff
@@ -28,10 +32,9 @@ const BetaHeader = () => {
       className="beta-header"
       alignItems={AlignItems.center}
     >
-      <Typography
-        variant={TypographyVariant.H7}
-        marginTop={0}
-        marginBottom={0}
+      <Text
+        variant={TextVariant.bodySm}
+        as="h6"
         className="beta-header__message"
         color={Color.warningInverse}
```

Comment:
```suggestion
        color={TextColor.warningInverse}
```
You can use this enum to define text color :)

Reply:
I have made the changes in the code. Could you please review it?
Thanks!